### PR TITLE
Mockbee Setup and created Homepage

### PR DESCRIPTION
**Pull request by anushasunil**
image

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import { Routes, Route } from "react-router-dom";
+import { Nav, Footer } from "../src/Components/index"
+import { Home, MockbeeHome, MockmanComponent} from "../src/pages/index"
```
Comment (dev-enforced) [inline src/App.js]: Instead of writing as `../src/pages/index` you can simply write it as `../src/pages`
Similarly for Line number 2

**Review by dev-enforced — APPROVED** [review]
The code structure looks good!!And the live preview as shown in the image also looks awesome. Awesome work @anushasunil !!

File: src/App.js (hunk 1 above)
Comment (anushasunil) [reply src/App.js]: ohh thanks for pointing this!

**Review by vanshitalavange — APPROVED** [review]
The UI looks great! Code looks clean and structured, good to merge

**Review by tanishq20 — APPROVED** [review]
Good to merge. Only mock backend setup in PR.

**Pull request merged**